CHANGELOG.md

```diff
@@ -89,7 +89,7 @@
 - PR #1497 Fix Thrust issue on CentOS caused by missing default constructor of host_vector elements
 - PR #1498 Add missing include guard to device_atomics.cuh and separated DEVICE_ATOMICS_TEST
 - PR #1506 Fix csv-write call to updated NVStrings method
-
+- PR #1510 Added nvstrings `fillna()` function
 
```

Comment:
Two line breaks between release versions

Reply:
Fixed in commit 74a9d260f1aea0f42b559bdf2d176a80fa563654